From: Kai-Chuan Hsieh <kaichuan.hsieh@canonical.com>
Date: Mon, 26 Dec 2022 10:24:20 +0800
Subject: sof-hda-dsp: Set Dmic0 Capture Switch on

Internal micrphone default is off after fresh installation.
Add operation to set the control on to align with other architecture.

Fixes: https://github.com/alsa-project/alsa-ucm-conf/pull/259
Signed-off-by: Kai-Chuan Hsieh <kaichuan.hsieh@canonical.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf b/ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf
index 791b26b..e1b9a56 100644
--- a/ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf
+++ b/ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf
@@ -83,6 +83,7 @@ If.dmic {
 		}
 		True.BootSequence [
 			cset "name='Dmic0 Capture Volume' 70%"
+			cset "name='Dmic0 Capture Switch' on"
 		]
 	}
 }
